bot/commands/latest.js

```diff
@@ -0,0 +1,19 @@
+const config = require('../../config.json');
+const Discord = require('discord.js');
+const request = require('request');
+
+module.exports.execute = async (client, message, args) => {
+    request('https://www.googleapis.com/youtube/v3/search?part=snippet&channelId=UCNmheQMf9sK1Axv3NCEhbSA&maxResults=1&q=%5BOinkcraft%5D&type=video&key=[REPLACE WITH YOUTUBE API KEY]', function (error, response, data) {
```

Comment:
This should be handled in the config instead of inline in the link, as Oinkbot will sync with the repository

Reply:
> This should be handled in the config instead of inline in the link, as Oinkbot will sync with the repository

I made some edits to the command. Is the change what you requested?